cli/lib/kontena/cli/plugins/uninstall_command.rb

```diff
@@ -6,9 +6,11 @@ class UninstallCommand < Kontena::Command
     include Kontena::Cli::Common
 
     parameter 'NAME', 'Plugin name'
+
     option "--force", :flag, "Force remove", default: false, attribute_name: :forced
 
     def execute
+      exit_with_error "Plugin #{pastel.cyan(name)} is not installed" unless Kontena::PluginManager.instance.installed(name) && !ENV['NO_PLUGINS']
```

Comment:
Does this strictly belong to this PR? Anyways, the `Kontena::PluginManager#uninstall_plugin` will already do the same check and raise an almost identical error, so this is redundant.